Light theme built for the app inside the file <style.js>

guillermoAMS opened the pull request:
Chequen si la barra cambió de color a púrpura. Si es así, cualquier componente que se agregue debería de seguir el mismo estilo definido por el tema que se hizo.

guillermoAMS committed:
Light theme built for the app inside the file <style.js>

gerardogalvez reviewed (approved):
Sí se ve púrpura.

gerardogalvez commented on flask-api/app.py:
```diff
@@ -18,6 +18,7 @@
 app.config['MYSQL_DATABASE_PASSWORD'] = os.getenv('MYSQL_DATABASE_PASSWORD')
 app.config['MYSQL_DATABASE_DB'] = os.getenv('MYSQL_DATABASE_DB')
 app.config['MYSQL_DATABASE_HOST'] = os.getenv('MYSQL_DATABASE_HOST')
+app.config['MYSQL_DATABASE_PORT'] = 3307
```
Éste que rollo?

guillermoAMS committed:
Updated file <app.py>

A line was added to solve an issue with the db connection. That line is now erased since it only works in my computer

osgonz commented on client/src/style/style.js:
```diff
@@ -0,0 +1,52 @@
+		},
+	},
+	typography: {
+		fontFamily: 		"Roboto",
```
Apparently Chrome tends to have issues when you choose Roboto as a font style without sans-serif. To keep the theme consistent with the body font-family set in style.css, you might want to switch line 38 for:
`fontFamily: 		["Roboto", "Helvetica", "Arial", 'sans-serif'],`

guillermoAMS committed:
Updated -fontFamily- in <Style.js>

There was an issue with fonts that didn't load in Chrome explorer. This is an update on the 'typography.fontFamily' object.
